presto-spi/src/main/java/com/facebook/presto/spi/Page.java

```diff
@@ -288,6 +288,11 @@ public Page prependColumn(Block column)
         return new Page(positionCount, result);
     }
 
+    public Page selectChannels(List<Integer> channels)
```

Comment:
@dain This would be useful in multiple places. For example, `com.facebook.presto.operator.WindowOperator#rearrangePage` and  `com.facebook.presto.operator.StreamingAggregationOperator#extractColumns`

Reply:
When I added it, I searched the code and couldn't find more places.  I'll add these when I update this PR.  If you know about other places, let me know.